CorsixTH/Lua/config_finder.lua

```diff
@@ -18,6 +18,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
+------------------------------------------------------CONFIG------------------------------------------------------
```

Comment:
Undo this change. The stylized section headers are only used within the config file.